src/jira/client/axios.ts

```diff
@@ -173,20 +162,24 @@ const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 		}
 		instrumentRequest(baseURL)(error?.response);
 
-		if (error.response?.status === 503 || error.response?.status === 405) {
+		if (error?.request?.method === "GET" && error?.response?.status === 405) {
+			/**
+			 * When sending a POST request to `https://OLD.atlassian.net/rest/deployments/0.1/bulk` or similar api on that jirahost,
+			 * it sends a 302 response, and redirects to a GET request `https://NEW.atlassian.net/rest/deployments/0.1/bulk`
+			 * This GET request is now failing with a 405 response, which is not an error on the app side
+			 * So treating this as a non-error case
```

Comment:
I was wondering if it makes sense to reconfigure axios not to follow redirects and to POST using the new hostname?...

Reply:
Yup, thought about that, make sense, but in this case, the jira host is redirected, not much point of still sending data to old jirahost.

Reply:
And if we follow the redirect and post, we will ends up sending twice the copy of data into new jira host, not sure what will that mean.